mempool/src/pool/store.rs

```diff
@@ -337,24 +348,53 @@ impl TxMempoolEntry {
         creation_time: Time,
     ) -> TxMempoolEntry {
         Self {
-            tx: WithId::new(tx),
             fee,
             parents,
             children: BTreeSet::default(),
             count_with_descendants: 1,
             creation_time,
             fees_with_descendants: fee,
+            size_with_descendants: tx.encoded_size(),
+            tx: WithId::new(tx),
         }
     }
 
+    pub(super) fn tx(&self) -> &WithId<Transaction> {
+        &self.tx
+    }
+
+    pub(super) fn fee(&self) -> Amount {
+        self.fee
+    }
+
     pub(super) fn count_with_descendants(&self) -> usize {
         self.count_with_descendants
     }
 
+    pub(super) fn fees_with_descendants(&self) -> Amount {
+        self.fees_with_descendants
+    }
+
+    pub(super) fn descendant_score(&self) -> DescendantScore {
+        (self.fees_with_descendants
+            / u128::try_from(self.size_with_descendants).expect("conversion"))
+        .expect("nonzero tx_size")
+        .into()
+    }
+
     pub(super) fn tx_id(&self) -> Id<Transaction> {
         WithId::id(&self.tx)
     }
 
+    pub(super) fn size(&self) -> usize {
+        // TODO(Roy) this should follow Bitcoin's GetTxSize, which weighs in sigops, etc.
```

Comment:
I'm confused. 🙃 Should this comment be addressed before the review or should it be changed to a regular TODO-comment?

Reply:
This is a regular TODO, it follows one of the formats suggested by Sam for long-standing todos, `TODO(name)`.